From: Gergely Nagy <algernon@madhouse-project.org>
Date: Sat, 29 Oct 2011 16:01:00 +0200
Subject: libmikmod-config manual page fix.

Remove a call to an undefined macro (.l) in the libmikmod-config
manual page.
---
 docs/libmikmod-config.1.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/libmikmod-config.1.in b/docs/libmikmod-config.1.in
index e0ef761..854e225 100644
--- a/docs/libmikmod-config.1.in
+++ b/docs/libmikmod-config.1.in
@@ -16,7 +16,7 @@ and link programs that use \fIlibmikmod\fP. It is also used internally
 to the .m4 macros for GNU autoconf that are included with \fIlibmikmod\fP.
 
 .SH OPTIONS
-.l
+
 \fIlibmikmod-config\fP accepts the following options:
 .TP 8
 .B  \-\-version
